src/rng.rs

```diff
@@ -393,14 +393,26 @@ impl Fill for [u8] {
     }
 }
 
-macro_rules! impl_fill {
+/// Implement `Fill` for given type `T`.
+///
+/// # Safety
+/// All representations of `[u8; size_of::<T>()]` are also representations of `T`.
```

Comment:
Replace 'representations of' with 'values representable by' and I'll agree with the sentence.

Reply:
I don't think we can say the values are the same, it really about the bytes by which they are represented in memory.